server/dash/dashboards/neighborhood.py

```diff
@@ -91,39 +131,67 @@ def update_council_figure(selected_council):
         color="type_name",
         color_discrete_sequence=DISCRETE_COLORS,
         labels=LABELS,
-        title="Request type trend for " + selected_council
+        title="Request type trend for " + selected_council,
     )
 
     fig.update_xaxes(
         tickformat="%a\n%m/%d",
     )
 
-    fig.update_traces(
-        mode='markers+lines'
-    )  # add markers to lines
+    fig.update_traces(mode="markers+lines")  # add markers to lines
 
     apply_figure_style(fig)
 
     return fig
 
 
-layout = html.Div([
-    html.H1(title),
-    dcc.Dropdown(
-        id='council_list',
-        clearable=False,
-        value="Arleta",
-        placeholder="Select a neighborhood",
-        options=populate_options()
-    ),
-    dcc.Graph(
-        id='graph1',
-        figure=fig,
-        config=CONFIG_OPTIONS
-    ),
-    dcc.Graph(
-        id='graph2',
-        figure=fig,
-        config=CONFIG_OPTIONS
-    )
-])
+layout = html.Div(
+    children=[
+        html.Div(
+            [
+                html.H1(title),
+                dcc.Dropdown(
+                    id="council_list",
+                    clearable=False,
+                    value="Arleta",
+                    placeholder="Select a neighborhood",
+                    options=populate_options(),
+                ),
+            ]
+        ),
+        html.Div(
+            children=[
+                dcc.Graph(id="graph1", figure=fig, config=CONFIG_OPTIONS),
+            ]
+        ),
+        html.Div(
+            [
+                html.Div(
+                    children=[
+                        html.Label("Moving Average Time Window:"),
+                    ],
+                    style={"width": "14%", "display": "inline-block"},
```

Comment:
Please leave a comment in the code describing how you chose 14% here. Ditto for the 15% below.

I'm not sure if this will have issues on mobile devices. We can check it after this code change is checked in.

Reply:
This was by trial and error. This needs to be reworked with UX

Reply:
Please leave a comment to that effect, so that when people read this code, they can understand what's stable and what is planned to be changed.